gdnative-core/src/nativescript/init/property/hint.rs

```diff
@@ -401,3 +412,58 @@ impl ColorHint {
         }
     }
 }
+
+/// Hints for Array, allowing nesting
+#[derive(Debug)]
+pub struct ArrayHint {
+    element_type: Option<Box<dyn Hint>>,
+}
+
+impl ArrayHint {
+    pub fn new() -> Self {
+        ArrayHint { element_type: None }
+    }
+
+    pub fn with_subhint<T>(hint: T) -> Self
+    where
+        T: Into<Box<dyn Hint>>,
+    {
+        ArrayHint {
+            element_type: Some(hint.into()),
+        }
+    }
+}
+
+impl Hint for ArrayHint {
+    #[inline]
+    fn export_info(&self) -> ExportInfo {
+        if let Some(element_type) = self.element_type {
+            let subinfo = element_type.export_info();
+            let hint_string = match (subinfo.variant_type, subinfo.hint_kind) {
+                (
+                    VariantType::VariantArray,
+                    sys::godot_property_hint_GODOT_PROPERTY_HINT_TYPE_STRING,
+                ) => format!("19:{}", subinfo.hint_string),
```

Comment:
Is this `19` value exported as a constant somewhere in the C API? If so, would it be better to use that constant instead of a hardcoded magic number?